ext/http/Cargo.toml

```diff
@@ -10,6 +10,9 @@ readme = "README.md"
 repository.workspace = true
 description = "HTTP server implementation for Deno"
 
+[features]
+"zombie_http_tracking" = []
```

Comment:
Why is this feature needed?

Reply:
If it's meant for debugging purpose I suggest to prefix it with double underscore

Reply:
Yeah, it's useful for debugging as the slab IDs don't necessarily have a lifetime that corresponds to a resource ID. Fixed.